### PR TITLE
Updated docs and scripts

### PR DESCRIPTION
**Pull request by Parth4337**
Updated scripts in deploy/testnet and changed default/home folder from $HOME to $HOME/.onomy
Changed docs/instructions accordingly put everything in $HOME/.onomy
All the binaries are now placed in $HOME/.onomy/bin 

Merged the 'setting-up-a-validator' and 'setting-up-a-validator-manual' as well as 'setting-up-afull-node' and 'setting-up-a-full-node-manual. added sections for both manual and scripted installations in same file.

Added RPM file to deploy/testnet and added instructions to install that file in pre-installation doc

### CODE REVIEW

File: docs/testnet/onomy-testnet-docs/setting-up-a-validator.md [hunk 1]
```diff
@@ -1,132 +1,201 @@
...
+
+Important Note: In the script, default file path is `$HOME/.onomy/onomy-testnet1/onomy`. If you have changed this path, than provide the home directory path accordingly in the `onomyd` command.
 
 ```
 bash peer-validator/makeValidator.sh
-
-Note: 1. Script will ask for enter faucet url get faucet token(Please enter http://testnet1.onomy.io:8000/)
 ```
+
+Script will ask for faucet url to get faucet token (Please enter http://testnet1.onomy.io:8000/)
```
Comment (dzmitryhil) [inline docs/testnet/onomy-testnet-docs/setting-up-a-validator.md:37]: can we set the default value "http://testnet1.onomy.io:8000/"  ?

File: docs/testnet/onomy-testnet-docs/setting-up-a-validator.md [hunk 2]
```diff
@@ -1,132 +1,201 @@
...
+
 You can check the validators of the Onomy chain by running:
 ```
 curl http//localhost:26657/validators
```
Comment (dzmitryhil) [inline docs/testnet/onomy-testnet-docs/setting-up-a-validator.md:41]: BTW, The same info you can get using cli:  "onomyd q staking validators"

File: deploy/testnet/peer-validator/makeValidator.sh [hunk 3]
```diff
@@ -32,7 +32,7 @@ GRAVITY_VALIDATOR_NAME=$(jq .validator_name $HOME/val_info.json | sed 's#\"##g')
 STAKE_DENOM="nom"
 #NORMAL_DENOM="samoleans"
```
Comment (dzmitryhil) [inline deploy/testnet/peer-validator/makeValidator.sh]: I guess we don't need this line

**Review by dzmitryhil — CHANGES_REQUESTED** [review]

File: deploy/testnet/peer-validator/makeValidator.sh (hunk 3 above)
Comment (Parth4337) [reply deploy/testnet/peer-validator/makeValidator.sh]: Removed

File: docs/testnet/onomy-testnet-docs/setting-up-a-validator.md (hunk 1 above)
Comment (Parth4337) [reply docs/testnet/onomy-testnet-docs/setting-up-a-validator.md:37]: Done

**Review by dzmitryhil — APPROVED** [review]

**Pull request merged**